.github/BOTMETA.yml

```
@@ -1119,6 +1119,13 @@ files:
   $plugins/inventory/scaleway.py: *scaleway
   $plugins/inventory/script.py:
     support: core
+  $plugins/inventory/vmware_vm_inventory.py: *vmware
```

Not tested, though you need something like;
```suggestion

   $plugins/inventory/vmware_vm_inventory.py:
     <<: *vmware
```

https://github.com/ansible/ansible/blob/a966662e9a9a8fd28832b2bc5a7d0d0df15b113d/.github/BOTMETA.yml#L460